test/scale/_test-data.scss

```diff
@@ -4,57 +4,118 @@
 @use '../../sass/internal/function' as f;
 @use '../../sass/tokens/function';
 @use '../../sass/scale/config';
+
 // Scale Tests
 // ===========
 
 // Config
 $test-sizes: (
   'root': 20px,
-  'text': '#root' ('convert-units': 'rem'),
-  'rhythm': '#text' (scale: '_fifth' 1),
-
-  'h1': '#text' (scale: 'linear' 4),
-  'h2': '#text' (scale: 'linear' 3),
-  'h3': '#text' (scale: 'linear' 2),
-
-  'big': '#text' (scale: '_fifth' 5),
-  'small': '#text' (scale: '_octave' -1),
+  'text': '#root'
+    (
+      'convert-units': 'rem',
+    ),
+  'rhythm': '#text'
+    (
+      scale: '_fifth' 1,
+    ),
+  'h1': '#text'
+    (
+      scale: 'linear' 4,
+    ),
+  'h2': '#text'
+    (
+      scale: 'linear' 3,
+    ),
+  'h3': '#text'
+    (
+      scale: 'linear' 2,
+    ),
+  'big': '#text'
+    (
+      scale: '_fifth' 5,
+    ),
+  'small': '#text'
+    (
+      scale: '_octave' -1,
+    ),
```

Comment:
I know I said parameters on their own line in an above comment, but this feels a lot less readable when we force each thing on a line here as opposed to what we had before. I don't know if anyone would agree with that, but throwing it out there incase we want to adjust something here. Thoughts @dvdherron @mirisuzanne?

Reply:
Yeah, I tends to agree with you when the map only has a single key/value -- but I like multiline when there's more than one. I'm not sure if a linter will let us be that specific? If I had to choose just one, I'd probably stick with the expanded version, and then ignore that rule for specific lines/files.

Reply:
@stacyk I agree that sass function parameters being on separate lines is generally more readable. If I had to choose, I would choose the stacked approach. 

I agree that the maps are a lot less readable in the way shown here ☝🏽 . If there's a rule to keep map keys/values on the same line that would be preferred.